python/torch_mlir_e2e_test/test_suite/stats.py

```diff
@@ -427,3 +427,116 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: VarDimKeepDimFalseModule())
 def VarDimKeepDimFalseModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4, 5))
+
+
+# ==============================================================================
+
+
+class VarCorrectionModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.var(x, dim=None, correction=2)
```

Comment:
Can you also add a test with `dim=[]`. This case should be the same in behavior as `dim=None`, and it would be nice to verify that that is the case in torch-mlir